content/en/serverless/glossary/_index.md

```diff
@@ -0,0 +1,91 @@
+| Edge location         | An AWS data center used to perform service-specific operations.                      |
+| Event | A JSON document that contains data for a Lambda function to process. |
+| Lambda function | A serverless function in Lambda. Each function had code to process events and can be invoked to run. |
+| Lambda layer | A ZIP archive that contains additional code—for example, libraries, a custom runtime, configuration files, or other dependencies. You can use Lambda layers to use libraries in your serverless functions without having to include these libraries in your deployment package. |
+| Managed policy | An IAM policy that can be attached to multiple users, groups, and roles. These can be created and managed by AWS or by a customer.|
+| Resource | An S3 bucket, EC2 instance, IAM user, or other entity that can be used in AWS. |
+| Resource property| When including a resource in an AWS CloudFormation stack, each resource can have one or more associated properties. |
+| Serverless Application Model (SAM)          | SAM is an Infrastructure as Code framework developed by AWS specifically focused on serverless applications.                                |
+
+
+
+### Datadog serverless for AWS Lambda concepts
+
+| Concept                         | Description                                                                                                                                                                                                          |
+|---------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Enhanced Lambda metrics | Enhanced Lambda metrics give you a view above and beyond the default Lambda metrics enabled with the AWS Lambda integration. These metrics are distinguished by being in the `aws.lambda.enhanced.*` namespace, and are Datadog’s best practice for setting real-time monitors on your serverless application health.|
+| Lambda library       | The Datadog Lambda Library is responsible for generating real-time enhanced Lambda metrics, submitting custom metrics, and enabling APM and Distributed Tracing for Node.js, Python, and Ruby                          |
+| Forwarder          | The Datadog Forwarder is an AWS Lambda function that ships logs, custom metrics, and traces from your environment to Datadog.                                |
+| Lambda extension          | The Datadog Lambda extension pushes real-time enhanced Lambda metrics, custom metrics, and traces from the Datadog Lambda Library to Datadog. It also forwards logs from your Lambda function to Datadog.                              |
```

Comment:
The Datadog Lambda Extension is a lightweight Datadog Agent that runs within the Lambda execution environment and ships serverless monitoring data to Datadog with minimal performance overhead.